db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/Scheduler.java

```diff
@@ -82,6 +87,8 @@ protected Scheduler(Clock clock, TaskRepository schedulerTaskRepository, TaskRep
         this.updateHeartbeatExecutor = Executors.newSingleThreadExecutor(defaultThreadFactoryWithPrefix(THREAD_PREFIX + "-update-heartbeat-"));
         SchedulerClientEventListener earlyExecutionListener = (enableImmediateExecution ? new TriggerCheckForDueExecutions(schedulerState, clock, executeDueWaiter) : SchedulerClientEventListener.NOOP);
         delegate = new StandardSchedulerClient(clientTaskRepository, earlyExecutionListener);
+        this.failureLogMethod = LogHelper.getLogMethod(LOG, logLevel);
+        this.logStackTrace = logStackTrace;
```

Comment:
We might make a bit more pretty if we introduce a wrapper-object, something like

```
failureLogger = new FailureLogger(LOG, logLevel, logStackTrace);
```
...
```
class FailureLogger
  logFailure(String message, Throwable, Object .. messageContext)
  ```

Reply:
Implemented.